### PR TITLE
utxo: Disable programmable pools

### PR DESCRIPTION
**Pull request by altonen**
Disable them for now at least as they're not part of the testnet

### CODE REVIEW

**Review by zorvan — APPROVED** [review]

**Review by muursh — APPROVED** [review]

**Review by iljakuklic — APPROVED** [review]

**Comment by altonen** [comment]
I'll wait until Anton and Carla have merged their PRs and then I'll merge this (and fix any conflicts)

**Pull request merged**
